openprescribing/media/js/measures.js

```diff
@@ -1,541 +1,161 @@
-        options.chart.height = 200;
-        options.legend.enabled = false;
-        if (_this.rollUpBy === 'org_id') {
-            ymax = _.max([localMax.y, _this.globalYMax.y]);
-            ymin = _.min([localMin.y, _this.globalYMin.y]);
-        } else {
-            var local90thMax = _.max(d.globalSeries['90'], _.property('y'));
-            ymax = _.max([localMax.y, local90thMax.y]);
-            var local90thMin = _.min(d.globalSeries['10'], _.property('y'));
-            ymin = _.min([localMin.y, local90thMin.y]);
-        }
-        var yAxisLabel = (isPercentageMeasure) ? '%' : 'Measure';
-        options.yAxis = {
-            title: {
-                text: yAxisLabel
-            },
-            max: ymax,
-            // If ymin is zero, Highcharts will sometimes pick a negative value
-            // for formatting reasons. Force zero as the lowest value.
-            min: _.max([0, ymin])
-        };
-        options.tooltip = {
-            formatter: function() {
-                // console.log('tooltip', this.point);
-                var num = Highcharts.numberFormat(this.point.numerator, 0),
-                    denom = Highcharts.numberFormat(this.point.denominator, 0),
-                    percentile = Highcharts.numberFormat(this.point.percentile, 0),
-                    str = '';
-                str += '<b>' + this.series.name;
-                str += ' in ' + Highcharts.dateFormat('%b \'%y',
-                                      new Date(this.x));
-                str += '</b><br/>';
-                if (!this.series.options.is_national_series) {
-                    str += d.numerator_short + ': ' + num;
-                    str += '<br/>';
-                    str += d.denominator_short + ': ' + denom;
-                    str += '<br/>';
-                }
-                str += 'Measure: ' +  Highcharts.numberFormat(this.point.y, 3);
-                str += (isPercentageMeasure) ? '%' : '';
-                if (!this.series.options.is_national_series) {
-                    //str += ' (' + num + '/' + denom + ')';
-                    str += ' (' + percentile + _this.getOrdinalSuffix(percentile) + ' percentile)';
-                }
-                return str;
-            }
-        };
-        return options;
-    },
-
-    getOrdinalSuffix: function(percentile) {
-        var lastChar = percentile.toString().slice(-1), suffix;
-        switch (lastChar) {
-            case '1':
-                suffix = (percentile == '11') ? 'th' : 'st';
-                break;
-            case '2':
-                suffix = (percentile == '12') ? 'th' : 'nd';
-                break;
-            case '3':
-                suffix = (percentile == '13') ? 'th' : 'rd';
-                break;
-            default:
-                suffix = 'th';
-        }
-        return suffix;
-    },
+                })
+                .fail(function(){
+                    console.log('data failed');
```

Comment:
If we're bothering to log something to the console, perhaps log details of the error? E.g. something like:

    .fail(function (jqXHR, textStatus, error) {
        console.log("Error " + error + " when making request " + jqXHR);
    });